nototools/unittests/font_tests.py

```diff
@@ -592,9 +592,9 @@ def setUp(self):
             if name in self.whitelist:
                 continue
             master_a[name].draw(pen_a)
-            area_a = pen_a.unload()
+            area_a = pen_a.value
```

Comment:
Oh, I didn't think about this. I think we need to actually create a new pen each time we draw a glyph (or add the unload method to the fonttools pen), otherwise the areas compound on one another.